components/badge.css

```diff
@@ -0,0 +1,51 @@
+/* resets browser defaults */
+@import "./reset.css";
+
+/* adds basic styling */
+@import "./style-guide.css";
+
+/* BADGE */
+
+/* common style */
+.badge {
+    background: var(--primary);
+    color: var(--background-grey);
+    display: inline-block;
+    padding: 0.25em 0.4em;
+    font-size: 75%;
+    font-weight: var(--bold);
+    border-radius: 0.25rem;
```

Comment:
use px if less than 10px

Reply:
I tried using px, but they need to be relative since the font-size is set to 75%.

Reply:
ok but be consistent I would suggest